### PR TITLE
osd/OSD: remove unused parameter osdmap_lock_name.

### PR DESCRIPTION
**Pull request by majianpeng**
Signed-off-by: Jianpeng Ma <jianpeng.ma@intel.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by majianpeng** [comment]
retest this please

File: src/osd/OSD.cc [hunk 1]
```diff
@@ -10340,8 +10340,7 @@ OSDShard::OSDShard(
     shard_name(string("OSDShard.") + stringify(id)),
     sdata_wait_lock_name(shard_name + "::sdata_wait_lock"),
     sdata_wait_lock{make_mutex(sdata_wait_lock_name)},
-    osdmap_lock_name(shard_name + "::osdmap_lock"),
-    osdmap_lock{make_mutex(osdmap_lock_name)},
+    osdmap_lock{make_mutex(shard_name +"::osdmap_lock")},
```
Comment (tchaikov) [inline src/osd/OSD.cc]: could you please add a space after "+" ?

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2020-01-10_01:36:54-rados-wip-kefu-testing-2020-01-10-0001-distro-basic-smithi/

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2020-01-10_01:36:54-rados-wip-kefu-testing-2020-01-10-0001-distro-basic-smithi/

standby failures are fixed by #32588

**Comment by tchaikov** [comment]
jenkins test make check

**Pull request merged**